### PR TITLE
Removes appframework dependancy from 3rdparty.

### PR DESCRIPTION
**Pull request by raghunayyar**
Removes Appframework dependency from 3rdparty.

@georgehrke lets get this one in. :D

### CODE REVIEW

**Comment by raghunayyar** [comment]
@BernhardPosselt I have given you credit on the angular directive here, though. Its the only part of appframework.js we are using.

File: js/app/directives/slidetoggleDirective.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+app.directive('ocClickSlideToggle', [
```
Comment (BernhardPosselt) [inline js/app/directives/slidetoggleDirective.js]: IIRC this is more or less in core now ;)

See data-apps-slide-toggle: https://doc.owncloud.org/server/8.1/developer_manual/app/css.html#settings-area

Comment (raghunayyar) [inline js/app/directives/slidetoggleDirective.js]: Awesome. I will check this asap and remove the directive if required.

Comment (georgehrke) [inline js/app/directives/slidetoggleDirective.js]: @raghunayyar Did you check yet?

**Comment by raghunayyar** [comment]
Updated as per @BernhardPosselt's call. This is awesome. @georgehrke please check and :+1:

**Comment by georgehrke** [comment]
:+1:

**Pull request merged**